### DISCUSSION TITLE
Comparison view

### DISCUSSION THREAD

**Discussion by zoe-jobson-IL** [artilleryio/artillery#3390] [Artillery]
Just a small comment - I love the comparison view and for many of our tests it will mean we don't have to send the metrics to datadog for comparison. One minor thing, though not sure if you can do anything about it. Sometimes, for exactly the same test - just with more users, or increased rate of starting the flow, the two views have one or two chart different - which mean that when you get to the bottom you are no longer comparing like with like. It would be good if a blank space could be inserted in those cases.

**Comment by manfromanotherland** [comment]
Hi @zoe-jobson-IL, glad you like it, thanks for the feedback! We will take a look and see if we can improve that aspect of it. In the mean time, could you share a screenshot of a comparison view where the charts don’t match for you? Would be useful to have that in mind when we work on a solution. Thanks!

**Reply by zoe-jobson-IL** [reply]
here https://app.artillery.io/vag1FEGIKUQy024fdc7gW7va/load-tests/tbx6h_mfwcen8fjj88ypzn8gqrn4aa493j8_qkkx/metrics?tags=name%3Aregister-and-baselining-test.yml&comparison=tfmnd_6jetzbp4ayqd487x9em67zjc6xfry_h85j

**Reply by manfromanotherland** [reply]
Hi @zoe-jobson-IL, just pushed some changes, now widgets will align with their pairs on the comparison view, hope that helps!

Here’s a little demo:

https://github.com/user-attachments/assets/371f7822-2486-4fed-b87a-2c4c65494d7a

**Reply by zoe-jobson-IL** [reply]
Yup just noticed you'd done that, much better Thank you 😁
